ocaml/idl/datamodel_pool.ml

```diff
@@ -1024,6 +1034,31 @@ let set_uefi_certificates =
       ]
     ~allowed_roles:_R_POOL_ADMIN ()
 
+let set_telemetry_next_schedule =
+  call ~name:"set_telemetry_next_schedule" ~lifecycle:[]
+    ~doc:
+      "Set the datetime after when the next round of telemetry collection will \
+       be carried out"
+    ~params:
+      [
+        (Ref _pool, "self", "The pool")
+      ; ( DateTime
+        , "value"
+        , "The datetime after when the next round of telemetry collection will \
+           be carried out"
+        )
+      ]
+    ~allowed_roles:_R_POOL_ADMIN ()
+
+let reset_telemetry_uuid =
+  call ~name:"reset_telemetry_uuid" ~lifecycle:[]
+    ~doc:
+      "Assign a new UUID to telemetry data. This can be done when the \
```

Comment:
```
Assign a new UUID to telemetry data. This will (intentionally) break the correlation
between previously collected telemetry data from this pool and future data.
```

Reply:
The purpose is not to break the correlation between previously collected data and future data. 
The correlation is for the association between a telemetry UUID and a pool. It should be kept inside the pool. If outside the pool, a telemetry UUID can be identified to belong to the pool, or even to the user, this is called a leaking of the correlation, which make the telemetry data not anonymous.
This API in this case allows user resetting the telemetry UUID of the pool. As a remedy, this only ensures the future telemetry data anonymous.

Reply:
How about this?
`Assign a new UUID to telemetry data. This will make the correlation between this pool and its future telemetry data unknown to outside of the pool again.`

Reply:
Maybe native speakers can comment. "Making a correlation unknown" feels more complicated than necessary to me but I'm not a native speaker.